packages/twenty-front/src/modules/ui/input/components/internal/date/components/InternalDatePicker.tsx

```diff
@@ -220,34 +221,70 @@ const StyledContainer = styled.div`
   }
 `;
 
+const StyledButtonContainer = styled.div`
+  align-items: center;
+  align-self: stretch;
+  display: flex;
+  gap: 8px;
+  height: 32px;
+  padding: 0px var(--Spacing-4px, 4px);
+`;
+
+const StyledButton = styled.button`
+  align-items: center;
+  background-color: transparent;
+  border: none;
+  color: gray;
+  cursor: pointer;
+  display: flex;
+  gap: 8px;
+  & .icon {
+    height: 16px;
+    width: 16px;
+  }
+`;
+
 export type InternalDatePickerProps = {
-  date: Date;
-  onMouseSelect?: (date: Date) => void;
+  date: Date | null;
+  onMouseSelect?: (date: Date | null) => void;
   onChange?: (date: Date) => void;
```

Comment:
We would need to have a new `clearable` props, because we don't want to allow clear everytime.

For the moment is should be false by default because otherwise it could introduce many bugs in the app.